src/timesync/timesyncd.c

```diff
@@ -83,7 +85,8 @@ static int load_clock_timestamp(uid_t uid, gid_t gid) {
         return 0;
 }
 
-int main(int argc, char *argv[]) {
+static int run(int argc, char *argv[]) {
+        DEFINE_DEFAULT_NOTIFY_MESSAGE_ON_EXIT;
```

Comment:
This means that the cleanup action (sending `STOPPING=1...`) will be performed e.g. if argument parsing fails. I don't think we want this.